api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
```

Comment:
There's two approaches you could take here:

The first is a regular expression approach, e.g. http://play.golang.org/p/Len2gWH_-x

The other uses the strings package: http://play.golang.org/p/Q_SVb8Gpnn